spark-statsd-reporter/src/main/java/org/apache/spark/metrics/sink/statsd/Configuration.java

```diff
@@ -1,25 +1,45 @@
 package org.apache.spark.metrics.sink.statsd;
 
 final class Configuration {
-    final static class Keys {
-        final static String HOST = "host";
-        final static String PORT = "port";
-        final static String PREFIX = "prefix";
-        final static String TAGS = "tags";
-        final static String POLL_INTERVAL = "poll.interval";
-        final static String POLL_UNIT = "poll.unit";
-        final static String RATE_UNIT = "rate.unit";
-        final static String DURATION_UNIT = "duration.unit";
+    public static final class Keys {
+
+        private Keys() {
+        }
+
+        public static final String HOST = "host";
```

Comment:
what' the reason for extending visibility? do we really need these fields being public?

Reply:
No need in particular @alembiewski suggested it would a best practice to make it `public`.